gobblin-audit/src/main/java/gobblin/audit/values/AliasUtils.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.audit.values;
+
+import java.lang.reflect.InvocationTargetException;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.reflect.ConstructorUtils;
+import org.reflections.Reflections;
+
+/**
+ * A utility methods to instantiate classes using their aliases.
+ * @see {@link HasAlias#getAlias()}
+ */
+@Slf4j
+public class AliasUtils {
```

Comment:
Please use instance methods. You can add a alias to classname cache which gets initialized in the constructor. Consider a different name like ClassAliasResolver or something along those lines.